newrelic/hooks/mlmodel_sklearn.py

```diff
@@ -79,13 +79,53 @@ def _nr_wrapper_method(wrapped, instance, args, kwargs):
         # _nr_wrapped attrs that will attach model info to the data.
         if method in ("predict", "fit_predict"):
             training_step = getattr(instance, "_nr_wrapped_training_step", "Unknown")
-            wrap_predict(transaction, _class, wrapped, instance, args, kwargs)
+            inference_id = uuid.uuid4()
+            wrap_predict(transaction, _class, inference_id, instance, args, kwargs)
```

Comment:
I wonder if we should rename `wrap_predict` to `create_feature_event`? Just so the code is more understandable. This model wrapping code is pretty bloated so anything we can do to make it more readable would be a big help I think.